integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/massindexing/MassIndexingFailureDefaultBackgroundErrorHandlerIT.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.mapper.orm.massindexing;
+
+import org.hibernate.search.mapper.orm.massindexing.MassIndexingFailureHandler;
+import org.hibernate.search.util.common.SearchException;
+import org.hibernate.search.util.impl.test.ExceptionMatcherBuilder;
+import org.hibernate.search.util.impl.test.rule.ExpectedLog4jLog;
+
+import org.junit.Rule;
+
+import org.apache.log4j.Level;
+
+public class MassIndexingFailureDefaultBackgroundErrorHandlerIT extends AbstractMassIndexingErrorIT {
```

Comment:
The name of this class looks wrong. It tests behavior when *errors* occur during mass indexing, with the default background *failure* handler. So it should be `MassIndexingErrorDefaultBackgroundFailureHandlerIT`.